CheckManagement/LocalRepository.cs

```
@@ -91,5 +168,53 @@ private void VerifyFolderPath()
         {
             Directory.CreateDirectory(FolderPath);
         }
+
+        // <summary>
+        // Deletes all checks from this repository.
+        // </summary>
+
+        /// <summary>
+        /// Deletes all checks from this repository.
+        /// </summary>
+        /// <returns>true if the files were successfully deleted. false otherwise.</returns>
+        public bool Clear()
+        {
+            try
+            {
+                var xmlFiles = Directory.EnumerateFiles(FolderPath, "*.xml");
+                foreach (var xmlFile in xmlFiles)
+                {
+                    File.Delete(xmlFile);
+                }
+                return true;
+            }
+            catch (Exception)
```

Instead of failing silently, it may be better to let the exception bubble up or printing a warning message. In case this produces errors that cause defects down the line.